app/utils/utils.js

```diff
@@ -0,0 +1,24 @@
+const properCase = (word) =>
```

Comment:
copy of https://github.com/ministryofjustice/hmpps-template-typescript/blob/main/server/utils/utils.ts